### PR TITLE
ttm: update for Leap 15.0

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

File: totest-manager.py [hunk 1]
```diff
@@ -37,6 +37,8 @@
 QA_FAILED = 2
 QA_PASSED = 3
 
+class NotFoundException(Exception):
+    True
```
Comment (jberry-suse) [inline totest-manager.py]: This is the use-case for `pass`.

File: totest-manager.py [hunk 2]
```diff
@@ -508,6 +518,44 @@ def write_version_to_dashboard(self, target, version):
...
+        url = self.api.makeurl(baseurl, query=query)
+        if self.dryrun:
+            logger.info("release %s/%s (%s)" % (project, package, set_release))
+        else:
+            self.api.retried_POST(url)
+
+        # XXX still legacy
+        for cd in self.livecd_products:
+            self._release_package('openSUSE:%s:Live' %
+                                 self.project, cd, set_release=release)
```
Comment (jberry-suse) [inline totest-manager.py]: nit: intended indent is off by one.

File: totest-manager.py [hunk 3]
```diff
@@ -408,24 +410,27 @@ def release_package(self, project, package, set_release=None):
...
-
+    def _release(self, set_release=None):
         for product in self.ftp_products:
-            self.release_package('openSUSE:%s' % self.project, product)
+            self._release_package('openSUSE:%s' % self.project, product)
 
         for cd in self.livecd_products:
-            self.release_package('openSUSE:%s:Live' %
+            self._release_package('openSUSE:%s:Live' %
                                  self.project, cd, set_release=release)
```
Comment (jberry-suse) [inline totest-manager.py]: nit: intended indent is off by one.

**Comment by lnussel** [comment]
$ pep8 --ignore=E501 totest-manager.py
$
:)

**Comment by nilxam** [comment]
:+1:

**Pull request merged**